packages/financial-templates-lib/src/clients/FinancialContractClient.js

```diff
@@ -337,3 +337,5 @@ class FinancialContractClient {
 }
 
 module.exports = { FinancialContractClient };
+
+// Test comment for CI test proposes
```

Comment:
Should we keep this comment?

Reply:
Removed the comment @nicholaspai, Thanks!